app/views/layouts/application.haml

```diff
@@ -11,14 +11,16 @@
       locals: { rosters: @current_user&.rosters || Roster.none }
     .container
       - if flash[:message].present?
-        .alert.alert-success
+        .alert.alert-success{ role: 'alert' }
           = flash[:message]
           - if flash[:change].present?
             (#{link_to 'undo', undo_change_path(flash[:change])})
       - if flash[:errors].present?
-        .alert.alert-danger
-          The following errors occurred:
-          %ul
-            - flash[:errors].each do |error_message|
```

Comment:
How about just `Array(flash[:errors]).each`? it won't do anything if it's already an array, but will then work if it isn't.

Reply:
I don't want to create a list for a single message.

Reply:
Okay, I usually try to cut down on lines but I suppose this method sort of cuts down on complexity.